clang/test/SemaSYCL/Inputs/sycl.hpp

```diff
@@ -218,6 +218,17 @@ class handler {
   }
 };
 
+class stream {
```

Comment:
I can add one... it makes the test that much longer.

I wrote this test as a part of this patch, since it actually manages to test all of the funny 'edge cases' I could think of to handle structs/arrays. It helped me find a handful of bugs in my implementation.

I can remove it from the patch if you'd like, I just thought it was a useful addition.

Reply:
But without this accessor we will have to figure out problems like we had with __init calls sequence again.

Reply:
Thats a good point, I can add an accessor here.